universal/router/matches/index.js

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+import WithSubRoutes from './WithSubRoutes'
+import { Redirect, Miss, Match, NavigationPrompt } from 'react-router'
+
+const match = {
+  Match,
+  Miss,
+  NavigationPrompt,
+  WithSubRoutes,
+  Redirect
+}
+
+export default match
```

Comment:
yes...see this example for a more "fleshed out" helper library for matches - https://github.com/spartansystems/foresight-client/tree/master/universal/router/matches/MatchHelpers